drms/client.py

```diff
@@ -682,7 +682,7 @@ def _generate_filenamefmt(self, sname):
             if si.keywords.loc[k].is_time:
                 pkfmt_list.append(f'{{{k}:A}}')
             else:
-                pkfmt_list.append(f'{k}')
+                pkfmt_list.append(f'{{{k}}}')
```

Comment:
My fault when I ported to the f strings. 

Checking the examples will be the way to check if this worked.